packages/fluxible-addons-react/connectToStores.js

```diff
@@ -53,7 +53,7 @@ function createComponent(Component, stores, getStateFromStores, customContextTyp
             }
         },
         render: function render() {
-            return React.createElement(Component, Object.assign({}, this.props, this.state, {ref: 'wrappedElement'}));
+            return React.createElement(Component, Object.assign({}, this.props, this.state, Component.prototype && Component.prototype.isReactComponent ? {ref: 'wrappedElement'} : null));
```

Comment:
Let's pull this out into a separate variable:

`var props = Component.prototype && Component.prototype.isReactComponent ? {ref: 'wrappedElement'} : null;`